js-exercises/alphabetic-shift/alphabeticShift.js

```diff
@@ -0,0 +1,9 @@
+const alphabeticShift = string => {
+  const shiftedString = [];
+  for(let char of string){
+    shiftedString.push( String.fromCharCode(char.charCodeAt() +1))
```

Comment:
Are you using a code formatter like, `prettier`? It'd have added a space between `+` and `1` 🤔